api/lib/application/campaigns/campaign-controller.js

```diff
@@ -138,6 +139,16 @@ module.exports = {
     return campaignAnalysisSerializer.serialize(campaignAnalysis);
   },
 
+  async findAssessmentParticipations(request) {
+    const campaignId = request.params.id;
+    const options = queryParamsUtils.extractParameters(request.query);
+
+    const userId = request.auth.credentials.userId;
```

Comment:
```suggestion
const requestResponseUtils = require('../../infrastructure/utils/request-response-utils');
/*...*/
const currentUserId = requestResponseUtils.extractUserIdFromRequest(request);
```
1) Renommage de userId en currentUserId (ça laisse pas entendre qu'on recherche les camp.parts. d'un utilisateur en particulier quand on lit en diagonal.
2) On passe par un helper existant pour récupérer le userId. En passant par `request.auth.credentials.userId`, j'ai l'impression d'exposer un fort couplage.